docs/pages/access-controls/access-request-plugins/ssh-approval-mattermost.mdx

```diff
@@ -280,10 +363,23 @@ output = "stderr" # Logger output. Could be "stdout", "stderr" or "/var/lib/tele
 severity = "INFO" # Logger severity. Could be "INFO", "ERROR", "DEBUG" or "WARN".
 ```
 </TabItem>
+<TabItem scopes={["cloud"]} label="Connect to the Proxy Service " options="Helm Chart">
+```yaml
+(!examples/resources/plugins/teleport-mattermost-helm-cloud.yaml!)
+```
+</TabItem>
+<TabItem scopes={["oss", "enterprise"]} label="Connect to the Auth Service " options="Helm Chart">
+```yaml
+(!examples/resources/plugins/teleport-mattermost-helm-self.yaml!)
+```
+</TabItem>
 </Tabs>
 
 ## Step 7/8. Test your Mattermost bot
 
+
+<Tabs dropdownCaption="Environment type" dropdownSelected="Executable">
```

Comment:
Since there are only two options here, I think we can use a regular `Tab`s component here and include labels for "Executable" and "Helm Chart", rather than dropdown options.